### PR TITLE
Run tests nightly against chia-blockchain main

### PR DESCRIPTION
**Pull request by hoffmang9**

### CODE REVIEW

**Comment by hoffmang9** [comment]
Note the successful run before removing on push - https://github.com/Chia-Network/chia-nft-minting-tool/actions/runs/6725699933/job/18280618336?pr=58

**Comment by hoffmang9** [comment]
Note also that I set this one to run 30 minutes later than all the rest. May be worth further staggering but it's at 4AM UTC so should be a slow CI time regardless

File: .github/workflows/run-tests-nightly.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+      - name: Setup Python environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
+      - name: Setup simulator
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
```
Comment (cmmarslender) [inline .github/workflows/run-tests-nightly.yml:39]: Shouldn't need to set this up twice

File: .github/workflows/run-tests-nightly.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    strategy:
+      max-parallel: 4
+      fail-fast: false
+      matrix:
+        os:
+          - runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```
Comment (cmmarslender) [inline .github/workflows/run-tests-nightly.yml:27]: If fixing the other, might as well update this to @v4 as well (latest version of the action)

**Review by cmmarslender — CHANGES_REQUESTED** [review]

File: .github/workflows/run-tests-nightly.yml (hunk 2 above)
Comment (hoffmang9) [reply .github/workflows/run-tests-nightly.yml]: Done

**Comment by hoffmang9** [comment]
nighlty flow passed here https://github.com/Chia-Network/chia-nft-minting-tool/actions/runs/6725844875/job/18281004608?pr=58

**Review by emlowe — APPROVED** [review]
looks good

**Review by cmmarslender — APPROVED** [review]

**Pull request merged**